src/cpu.jakt

```diff
@@ -1,5 +1,14 @@
 import system { System }
 
+// Sign extend byte
+function sxb(value: u8) -> u16 {
```

Comment:
I'm not precious about this begin a free function. Maybe it could be a method of CPU along side `.branch`?